modifiers/allocation/modifier.py

```diff
@@ -322,6 +322,7 @@ def slurm_instructions(self, v):
         sbatch_directives = list(f"#SBATCH {x}" for x in (srun_opts + sbatch_opts))
 
         v.mpi_command = f"srun {' '.join(srun_opts)}"
+        v.single_rank_mpi_command = "srun -n 1 -N 1"
```

Comment:
@scheibelp I've added a single_rank_mpi_command that is used for mpi executable meant to be run on a single node